tedx-website/src/components/speakerlist.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PageTitle from "../page_title/page_title";
+import speakersData from "../../data/speakers_data"
+import Speaker from "../speaker/speaker"
+
+function SpeakerList(props) {
+    const speakers = speakersData;
+
+    return (
+        <div className="speakers-page page-container">
+            <PageTitle title="Speakers" />
+            {props.map(speakers => (
```

Comment:
nit: can we use `speaker` instead of `speakers` since each of these elements is one speaker?